processor/src/main/java/org/mapstruct/ap/internal/model/assignment/GetterWrapperForCollectionsAndMaps.java

```diff
@@ -41,20 +40,17 @@
     /**
      * @param decoratedAssignment
      * @param thrownTypesToExclude
-     * @param existingVariableNames
      * @param targetType
      * @param fieldAssignment
      */
     public GetterWrapperForCollectionsAndMaps(Assignment decoratedAssignment,
```

Comment:
I noticed that in the `AdderWrapper` and `SetterWrapper` you renamed the `decoratedAssignment` to `rhs`. Should we do the renaming everywhere (the other `AssignmentWrapper`(s), and the `AssignmentWrapper` itself), or it does not make sense?